projects/plutus/inc/current_sense.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
+
+// Current sense module for the LTC2484 ADC.
+// Requires gpio, interrupts, and soft timers to be initialized
```

Comment:
side effects and usage please

potential faults etc

Reply:
Documentation... truly a worthy final boss